Hotfix/1.0.1 (#2)
polyphene/getting-started: README.md

> Contains necessary assets to go along the tutorial example:
> - `sources` contains data sources tht will be used in the tutorial.

Reviewer: ```suggestion
Contains necessary assets like data sources that go along the tutorial.
```

(rephrasing and fixing a typo)
